tests/handlers/test_combined_data.py

```diff
@@ -344,9 +349,104 @@ def test_turnout_factor_as_non_predictive(va_governor_county_data):
     turnout_factor_lower = 0.95
     turnout_factor_upper = 1.2
     (_, _, unexpected_data) = combined_data_handler.get_units(
-        100, turnout_factor_lower, turnout_factor_upper, ["county_fips"]
+        100, turnout_factor_lower, turnout_factor_upper, 0.4, [], [], ["county_fips"]
     )
     over = combined_data_handler.data[combined_data_handler.data.turnout_factor >= turnout_factor_upper].shape[0]
     under = combined_data_handler.data[combined_data_handler.data.turnout_factor < turnout_factor_lower].shape[0]
     assert unexpected_data.shape[0] == over + under
     assert len(unexpected_data[unexpected_data["unit_category"] == "non-modeled"]) == over + under
+
+
+def test_margin_change_threshold_as_non_predictive(va_governor_county_data):
+    election_id = "2017-11-07_VA_G"
+    office = "G"
+    geographic_unit_type = "county"
+    estimands = ["margin"]
+
+    live_data_handler = MockLiveDataHandler(
+        election_id, office, geographic_unit_type, estimands, data=va_governor_county_data
+    )
+    current_data = live_data_handler.get_n_fully_reported(n=133)
+    va_governor_county_data["baseline_weights"] = (
+        va_governor_county_data.baseline_dem + va_governor_county_data.baseline_gop
+    )
+    va_governor_county_data["baseline_margin"] = (
+        va_governor_county_data.baseline_dem - va_governor_county_data.baseline_gop
+    )
+    va_governor_county_data["baseline_normalized_margin"] = (
+        va_governor_county_data.baseline_margin / va_governor_county_data.baseline_weights
+    )
+    va_governor_county_data["last_election_results_margin"] = va_governor_county_data.baseline_margin
+
+    combined_data_handler = CombinedDataHandler(va_governor_county_data, current_data, estimands, geographic_unit_type)
+
+    margin_change_threshold = 0.1
+    (reporting_data, _, unexpected_data) = combined_data_handler.get_units(
+        100, 0.2, 2.5, margin_change_threshold, [], [], ["county_fips"]
+    )
+    over = combined_data_handler.data[
+        combined_data_handler.data.normalized_margin_change > margin_change_threshold
+    ].shape[0]
+    assert unexpected_data.shape[0] == over
+    assert len(unexpected_data[unexpected_data["unit_category"] == "non-modeled"]) == over
+    assert reporting_data.shape[0] == 133 - over
+
+
+def test_unit_blacklist(va_governor_county_data, rng):
```

Comment:
I'm fine with either "blocklist" or "blacklist", so can you either change this to "blocklist" or change `CombinedDataHandler` to "blacklist"? 🤔

Reply:
yes, absolutely. I thought I had found them all, but I guess not

Reply:
should be done